docs/pages/operator/monitoring.mdx

```diff
@@ -23,6 +23,30 @@ kube-system   coredns-854c77959c-q5878          3m           17Mi
 kube-system   metrics-server-5fbdc54f8c-fgrqk   0m           6Mi
 ```
 
+If you see below error after installing metrics-server [https://github.com/k3s-io/k3s/issues/5344],
+```
+loading OpenAPI spec for "v1beta1.metrics.k8s.io" failed with: failed to retrieve openAPI spec, http error: ResponseCode: 503
+```
+you can take the steps given below,
+- metrics_patch.yaml
```

Comment:
Rename to `create a file named 'metrics_patch.yaml' with the following contents:`